### PR TITLE
Stop generating gh-pages during release, update message

### PR DESCRIPTION
**Pull request by zpao**
We don't push the gh-pages branch directly anymore so get rid of that code from our release process.

### CODE REVIEW

File: grunt/tasks/release.js [hunk 1]
```diff
@@ -103,39 +102,16 @@ function bower() {
   _gitCommitAndTag(BOWER_PATH, VERSION_STRING, VERSION_STRING, done);
 }
 
-function docs() {
-  var done = this.async();
-
-  grunt.file.copy('build/react-' + VERSION + '.zip', 'docs/downloads/react-' + VERSION + '.zip');
-  grunt.file.copy('build/react.js', 'docs/js/react.js');
```
Comment (spicyj) [inline grunt/tasks/release.js]: Isn't this part still important?

Comment (zpao) [inline grunt/tasks/release.js]: … yea, it is. oops

File: grunt/tasks/release.js [hunk 2]
```diff
@@ -134,8 +107,10 @@ function msg() {
   var steps = [
     'Still todo:',
     '* put files on CDN',
+    '* add starter pack (git add -f docs/downloads/react-version.zip)',
```
Comment (spicyj) [inline grunt/tasks/release.js]: You also need to commit, right?

Comment (zpao) [inline grunt/tasks/release.js]: Yea but I figured the people doing this (me and you) are smart enough to know that. This isn't meant to be a complete step by step but a gentle reminder

**Pull request merged**